src/outfit.ts

```diff
@@ -590,16 +638,20 @@ export class Outfit {
     const modes = convertToLibramModes(this.modes);
     if (this.modifier) {
       const allRequirements = [
-        new Requirement([this.modifier], {
+        new Requirement(this.modifier, {
           preventSlot: [...usedSlots],
           preventEquip: this.avoid,
           modes: modes,
+          bonusEquip: this.bonuses,
         }),
       ];
       if (extraOptions) allRequirements.push(new Requirement([], extraOptions));
 
       if (!Requirement.merge(allRequirements).maximize()) {
-        throw `Unable to maximize ${this.modifier}`;
+        cliExecute("refresh inventory");
```

Comment:
In what state does a failed maximizer call leave the KoL character? Do we have to rerun the forced equipment above as well?

Reply:
Good question, I'm not actually sure. Typically I've found that failed maximizer calls either happen when mafia starts thinking I have, say, two june cleavers, or when it fails to meet a boolean requirement. I'm yet to personally encounter a situation where it starts to touch slots we've banned.

Reply:
I'll go ahead and run through that process again regardless